response/core/models/timeline.py

```diff
@@ -1,22 +1,25 @@
 from datetime import datetime
 from django.db import models
-from django.contrib.postgres.fields import JSONField
+from jsonfield import JSONField
 
 from response.core.models.incident import Incident
 
 
-class TimelineEvent(models.Model):
+EVENT_TYPES = (
+    ("text", "Freeform text field"),
+    ("slack_pin", "A message pinned from Slack"),
+)
+
 
-    EVENT_TYPES = (("text", "Freeform text field"),)
+class TimelineEvent(models.Model):
 
     incident = models.ForeignKey(Incident, on_delete=models.CASCADE)
     timestamp = models.DateTimeField(
         null=False, default=datetime.now, help_text="Time of when this event occurred."
     )
     text = models.TextField(help_text="Freeform text describing this event")
-    event_type = models.CharField(
-        max_length=10, choices=EVENT_TYPES, help_text="Type of event."
```

Comment:
Do we no longer want to restrict it to this enum?

Reply:
Nope. I realised that if people (including Monzo) want to add new custom event types to their private projects, they wouldn't be able to without forking this